dev-qt/qtsql/qtsql-5.7.1-r1.ebuild

```diff
@@ -5,7 +5,7 @@ EAPI=6
 QT5_MODULE="qtbase"
 inherit qt5-build
 
-DESCRIPTION="SQL abstraction library for the Qt5 tooolkit"
+DESCRIPTION="SQL abstraction library for the Qt5 toolkit"
```

Comment:
Use "framework" instead of "toolkit", that's upstream's preferred wording since Qt 5, and what we use everywhere else.